luaui/RmlWidgets/gui_tech_points/gui_tech_points.lua

```diff
@@ -299,6 +299,10 @@ local function updateUI()
 	end
 end
 
+local function unitDefBlocked(unitDefID, teamID)
+	Spring.Echo("Widget received unitDefBlocked:", unitDefID, teamID)
```

Comment:
consider whether widgets should know Reasons why stuff is blocked (they probably do, so that the player can distinguish "blocked due to campaign progress" vs "shipyard blocked because this is a water map" vs "blocked due to modoptions" vs ...")